libs/common/src/types/fee-record-correction.ts

```diff
@@ -1,5 +1,17 @@
+import { FeeRecordStatus } from './utilisation-reports';
+
 export type RequestedByUser = {
   id: string;
   firstName: string;
   lastName: string;
 };
+
+export type FeeRecordCorrection = {
```

Comment:
We are going to have quite a few types that differ a little related to fee records, perhaps we should call this `FeeRecordCorrectionSummary` to help avoid any future overlaps since this doesn't return much data as opposed to the screen once they click into a correction to see the full correction details

Reply:
changed